### PR TITLE
Refactor/methods

### PR DESCRIPTION
**Pull request by coopterrones**
## What is the change?
naming conventions in all tests
## What does it fix?
naming conventions in tests 
## Is this a fix or a feature? 
Fix
## Where should the reviewer start?
* line 105 activity.js
* line 68 sleep.js
* line 211 activity-test.js
* line 138 hydration-test.js
* line 343 sleep-test.js
* line 49 user-repo-test.js
## How should this be tested?
* check all tests for passing
* all naming conventions should be updated and correlate to the implementation code 
@brycemara @dietza @coopterrones

### CODE REVIEW

File: src/Activity.js [hunk 1]
```diff
@@ -102,10 +102,7 @@ class Activity {
     let data = this.activityData;
     let sortedUserArray = (userRepo.makeSortedUserArray(id, data)).reverse();
     let streaks = sortedUserArray.reduce((streakDays, currentDay, index) => {
-      let qualifyingDay;
-      if (index >= 2) {
-        qualifyingDay = (sortedUserArray[index - 2][relevantData] < sortedUserArray[index - 1][relevantData] && sortedUserArray[index - 1][relevantData] < sortedUserArray[index][relevantData]);
-      };
+      let qualifyingDay = this.isQualifyingDay(sortedUserArray, index, relevantData);
```
Comment (dietza) [inline src/Activity.js:105]: Glad we got some clarification in the check-in and separated this complicated logic out - this seems friendlier at least!

File: src/Sleep.js [hunk 2]
```diff
@@ -65,16 +65,26 @@ class Sleep {
   };
 
   getWinnerNamesFromList(sortedArray, userRepo) {
+    let bestSleepers = this.getBestSleepers(sortedArray);
```
Comment (brycemara) [inline src/Sleep.js:68]: May need to add some more tests to account for the helper functions created here

**Comment by brycemara** [comment]
Glad the tests are up and running again. Re-naming complete!

**Review by dietza — APPROVED** [review]
Looked at this together and changes are great (tests are updated with new naming and pass again!)
Can return later to add additional tests where methods were broken up.

**Pull request merged**